src/gui/src/drcWidget.cpp

```diff
@@ -466,6 +467,10 @@ void DRCWidget::loadTRReport(const QString& filename)
                      line);
     }
 
+    if (type == "Horizontal congestion" || type == "Vertical congestion") {
+      is_congestion_file = true;
+    }
```

Comment:
Rather than make this congestion specific just allow a layer value like "-" for no value